Tests/Api/Report/BaseReportTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+
+namespace Tests\Werkspot\BingAdsApiBundle\Tests\Api\Report;
+
+
+use BingAds\Reporting\ReportFormat;
+use BingAds\Reporting\ReportLanguage;
+use BingAds\Reporting\ReportRequest;
+use Werkspot\BingAdsApiBundle\Api\Report\BaseReport;
+
+class BaseReportTest extends \PHPUnit_Framework_TestCase
+{
+    public function testSetFormat()
+    {
+        $report = new BaseReport();
+        $this->assertNull($report->getRequest()->Format);
+
+        $report->setFormat(ReportFormat::Csv);
```

Comment:
What is 'setFormat', there are no comments in the implementation, can it be anything? object, array, int, bool? if it can be speccific values only we should use enum, but at least add a comment